src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -2163,4 +2249,13 @@ void PolicyHandler::OnUpdateHMILevel(const std::string& device_id,
   }
   UpdateHMILevel(app, level);
 }
+
+template <typename Func, typename... Args>
```

Comment:
@SNiukalov I think you could merge this template definition with declaration and keep whole in header file